pkg/capacity/manager_test.go

```diff
@@ -390,25 +211,27 @@ func TestGetOptimalTargetClusterInBatchesBy50WhenTwoClusterHaveTheSameUsage(t *t
 		t.Run(fmt.Sprintf("for the given limit of max number of spaces per cluster: %d", limit), func(t *testing.T) {
 			for _, numberOfSpaces := range []int{0, 8, 50, 88, 100, 123, 555} {
 				t.Run(fmt.Sprintf("when there is a number of spaces at the very beginning %d", numberOfSpaces), func(t *testing.T) {
-					toolchainStatus := NewToolchainStatus(
-						WithMetric(toolchainv1alpha1.MasterUserRecordsPerDomainMetricKey, toolchainv1alpha1.Metric{
-							string(metrics.Internal): 1000,
-						}),
-						WithMetric(toolchainv1alpha1.UserSignupsPerActivationAndDomainMetricKey, toolchainv1alpha1.Metric{
-							"1,internal": 1000,
-						}),
-						WithMember("member1", WithSpaceCount(1000), WithNodeRoleUsage("worker", 68), WithNodeRoleUsage("master", 65)),
-						WithMember("member2", WithSpaceCount(numberOfSpaces), WithNodeRoleUsage("worker", 55), WithNodeRoleUsage("master", 60)),
-						WithMember("member3", WithSpaceCount(numberOfSpaces), WithNodeRoleUsage("worker", 55), WithNodeRoleUsage("master", 50)))
-
 					// member2 and member3 have the same capacity left and the member1 is full, so no one can be provisioned there
-					spc1 := hspc.NewEnabledValidTenantSPC("member1")
-					spc2 := hspc.NewEnabledValidTenantSPC("member2", spc.MaxNumberOfSpaces(limit))
-					spc3 := hspc.NewEnabledValidTenantSPC("member3", spc.MaxNumberOfSpaces(limit))
-
-					fakeClient := commontest.NewFakeClient(t, toolchainStatus, spc1, spc2, spc3)
-					InitializeCounters(t, toolchainStatus)
-					clusterBalancer := capacity.NewClusterManager(commontest.HostOperatorNs, fakeClient)
+					spc1 := hspc.NewEnabledValidTenantSPC("member1",
+						spc.MaxNumberOfSpaces(1000),
+						spc.WithConsumedSpaceCount(1000),
+						spc.WithConsumedMemoryUsagePercentInNode("worker", 68),
+						spc.WithConsumedMemoryUsagePercentInNode("master", 65))
+					spc2 := hspc.NewEnabledValidTenantSPC("member2",
+						spc.MaxNumberOfSpaces(limit),
+						spc.WithConsumedSpaceCount(numberOfSpaces),
+						spc.WithConsumedMemoryUsagePercentInNode("worker", 55),
+						spc.WithConsumedMemoryUsagePercentInNode("master", 60))
+					spc3 := hspc.NewEnabledValidTenantSPC("member3",
+						spc.MaxNumberOfSpaces(limit),
+						spc.WithConsumedSpaceCount(numberOfSpaces),
+						spc.WithConsumedMemoryUsagePercentInNode("worker", 55),
+						spc.WithConsumedMemoryUsagePercentInNode("master", 50))
+
+					status := test.NewToolchainStatus()
+					test.InitializeCounters(t, status)
```

Comment:
the same also here and at other places, don't use the utilization from the SPC, but rather from the counter instead